README.rst

```diff
@@ -41,6 +41,10 @@ From the command-line, this requires some configuration::
     endpoint = https://api.exoscale.ch/compute
     key = cloudstack api key
     secret = cloudstack api secret
+    # Optional ca authority certificate
+    verify = /path/to/certs/exoscale_ca.crt
+    # Optional client PEM certificate
+    cert = /path/to/client_exoscale.pem
```

Comment:
Changed README to not explicitly mention verify = False, but it is still an option. Do you still object to this option?